tensorlayer/files.py

```diff
@@ -722,6 +823,8 @@ def load_flickr1M_dataset(tag='sky', size=10, path="data", n_threads=50, printab
     >>> images = tl.files.load_flickr1M_dataset(tag='zebra')
 
     """
+    import shutil
```

Comment:
we should almost always put `import` on top of file:
https://stackoverflow.com/questions/1188640/good-or-bad-practice-in-python-import-in-the-middle-of-a-file
`shutil` is a standard module in python, it should be cheap to import.